spec/transactionSpec.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var Transaction = require('../src/transaction').Transaction
+var convertStringToDate = require('../src/transaction').convertStringToDate
+
+describe('Transaction', function () {
+  var transaction, date, amount, dateConverted;
+
+  beforeEach(function () {
+    date = '10/01/2012'
+    dateConverted = convertStringToDate(date)
+    amount = 100;
+    transaction = new Transaction(amount, date);
+  });
+
+  describe('is initialized', function () {
+    it('with a date', function () {
+      expect(transaction.date.getTime()).toBe(dateConverted.getTime());
+    });
```

Comment:
It's idiomatic for there to be a line of whitespace between each describe and it.